_sass/jekyll-theme-eclipsefdn.scss

```diff
@@ -20,3 +20,15 @@ td {
     border: 1px solid #ebebeb;
 }
 
+#custom-search-form {
```

Comment:
Why the need of custom css for the search? Why do we need this here but not on eclipse.org?

Reply:
I'm adding custom CSS because on eclipse.org we rely on styles coming from Google when using their search widget. Since we are now using a simple form here, we need to add some extra CSS.

Reply:
The whole point of this transition is to not include files from google.

I think this is something we need to fix in solstice assets not here.